Vital check code/Checker.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VitalAssignment
+{
+    class Checker
+    { 
+        public List<Vital> VitalList = new List<Vital>()
+                                        {new Vital("bpm", 70, 150),
+                                         new Vital("spo2", 90, 100),
+                                         new Vital("respRate", 30, 95)};
+ 
+         public bool vitalsAreOk(int[] VitalValues,
+                                 VitalAlert typeOfAlert)
+        {
+             bool[] a = new bool[2] {true, true};
+            for (int i=0; i < VitalList.Count; i++)
```

Comment:
Any reason to have an array of two bools?

Reply:
I wanted to have two bool variables, one to check the return value of isVitalOk() and other to return the value of vitalsAreOk(). I tried using one variable for both checking and return but it was showing error in one of the test cases.

Reply:
You could have only one bool corresponding to vitalsAreOk, and place the expression itself as the condition.
Like:
```C#
if (VitalList[i].isVitalOk(VitalValues[i], typeOfAlert) == false)
{
  overallVitalsOk = false;
}
```